Framework/Algorithms/test/ModeratorTzeroTest.h

```diff
@@ -244,9 +244,11 @@ class ModeratorTzeroTest : public CxxTest::TestSuite {
         Mantid::Kernel::UnitFactory::Instance().create("TOF");
     BinEdges xdata(numBins + 1, LinearGenerator(0.0, 4.0));
     const double peakHeight(1000.), peakCentre(7000.), sigmaSq(1000 * 1000.);
+
+    auto &Y = testWS->mutableY(0);
     // tof ranges from 0 to 16000 (units assumed micro-seconds
     for (int ibin = 0; ibin < numBins; ++ibin) {
-      testWS->dataY(0)[ibin] =
+      Y[ibin] =
           peakHeight * exp(-0.5 * pow(xdata[ibin] - peakCentre, 2.) / sigmaSq);
```

Comment:
It must be the 10th time I see this gaussian somehwhere, maybe we should add a `GaussianGenerator`, or some other way to generator this?